packages/bsky/src/views/index.ts

```diff
@@ -1426,6 +1429,14 @@ export class Views {
       recordInfo = state.reposts?.get(notif.uri)
     } else if (uri.collection === ids.AppBskyGraphFollow) {
       recordInfo = state.follows?.get(notif.uri)
+    } else if (uri.collection === ids.AppBskyGraphVerification) {
+      recordInfo = {
+        record: {
+          $type: 'app.bsky.notification.defs#recordDeleted',
+        },
+        // Pre-computed CID for the dummy record above.
+        cid: 'bafyreidad6nyekfa4a67yfb573ptxiv6s7kyxyg2ra6qbbemcruadvtuim',
+      }
```

Comment:
I might recommend pulling these values out into constants.  Also, in the case of a verification I think we'd want the details of the verification record to appear here.